jetstream/workflows/run.yaml

```diff
@@ -48,6 +48,11 @@ spec:
         "--dataset_id={{workflow.parameters.dataset_id}}", 
         "--project_id={{workflow.parameters.project_id}}"
       ]
+      resources:
+        requests:
+          memory: 8Gi   # make sure there is at least 8Gb of memory available for the task
```

Comment:
Most tasks use around 1-2GB of memory but there are some that use up to 12GB. With this the Kubernetes scheduler makes sure that the job will have at least 8GB of memory available when running. We currently have up to 3 nodes of 64GB available with auto-scaling, so jobs will get assigned to additional nodes to fulfill the memory requirements.